plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/intentions/ConvertConcatenationToBuildStringIntention.kt

```diff
@@ -0,0 +1,54 @@
+// Copyright 2000-2021 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import org.jetbrains.kotlin.idea.KotlinBundle
+import org.jetbrains.kotlin.idea.core.ShortenReferences
+import org.jetbrains.kotlin.idea.core.replaced
+import org.jetbrains.kotlin.psi.*
+
+class ConvertConcatenationToBuildStringIntention : ConvertToStringTemplateIntention() {
```

Comment:
Please, don't inherit from `ConvertToStringTemplateIntention`. `ConvertToStringTemplateIntention` is absolutelly different intention which is not related to `ConvertConcatenationToBuildStringIntention`. Either choose more common ancestor for inheritance or extract common code from `ConvertConcatenationToBuildStringIntention`